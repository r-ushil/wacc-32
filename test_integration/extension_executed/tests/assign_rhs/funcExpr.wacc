# function calls are not first-class expressions

# Output:
# #syntax_error#

# Exit:
# 0

# Program:

begin
  int f() is
    return 0 
  end
  int x = f() + f();

  # should output 0
  println x
end

# UPDATE: THIS IS NOW A VALID TEST CASE
