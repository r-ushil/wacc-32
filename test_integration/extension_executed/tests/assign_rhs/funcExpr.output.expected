Compiler exit code: 0
--------- stdout ---------
0
--------------------------
Exit code: 0
